bin/plugin/commands/performance.js

```diff
@@ -31,6 +31,24 @@ const RESULTS_FILE_SUFFIX = '.performance-results.json';
  * @property {string=}  wpVersion   The WordPress version to be used as the base install for testing.
  */
 
+const fAccent = formats.success;
+const fTitle = formats.title;
+const fWarning = formats.warning;
+
+/**
+ * A logging helper for printing steps and their substeps.
+ *
+ * @param {number} sub  Value to indent the log.
+ * @param {any}    msg  Message to log.
+ * @param {...any} args Rest of the arguments to pass to console.log.
+ */
+function logStep( sub, msg, ...args ) {
+	const indent = '    ';
+	const prefix = sub === 0 ? '▶\u00A0' : '>\u00A0';
+	const newline = sub === 0 ? '\n' : '';
+	return log( newline + indent.repeat( sub ) + prefix + msg, ...args );
+}
```

Comment:
I like this helper, but I don't know what `sub` is after reading it a few times. is there a clearer name or description we can use?

Reply:
I guess `indent` would be more descriptive here?

Reply:
Maybe `subLevel`?

Reply:
Changed to suggested `logAtIndent( indent, ...` in https://github.com/WordPress/gutenberg/pull/54309/commits/52d04961ca96f2dd94dec02080a4cfc227a3e0a1